storage/jellyfish_merkle/src/mock_tree_store.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    node_type::{Node, NodeKey},
+    RetiredStateRecord, TreeReader, TreeUpdateBatch,
+};
+use failure::prelude::*;
+use std::{
+    collections::{hash_map::Entry, BTreeSet, HashMap},
+    sync::RwLock,
+};
+use types::transaction::Version;
+
+#[derive(Default)]
+pub(crate) struct MockTreeStore(RwLock<(HashMap<NodeKey, Node>, BTreeSet<RetiredStateRecord>)>);
```

Comment:
I don't mind the MockTreeStore tuple wrapping the RwLock, but having a dual tuple looks weird.  Since this is just for mocking I won't complain though.